apps/test-pololu370-encoder/src/main.cpp

```diff
@@ -0,0 +1,46 @@
+#include "Logger.h"
+#include "Pololu37D.h"
+
+void read_all();
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Pololu37D::Config config = {PA_5, PA_6, NC, 0, GPIO::QEI::Encoding::X2_ENCODING};
```

Comment:
I think we should be using the X4_ENCODING scheme. From the [encoder link](https://www.pololu.com/product/2821) the 64 counts / rev is based on X4 encoding:
> The quadrature encoder provides a resolution of 64 counts per revolution of the motor shaft when counting both edges of both channels.

Maybe this is why the measurements you're getting are off by a factor of 2?

Reply:
Ah that makes sense, I had assumed that we were supposed to use X2 encoding by default. Should I switch over to X4?

Reply:
Yes try that and make sure to test it again

Reply:
changed it to X4 encoding and its working as intended